### ISSUE TITLE
Terrain shader bug?

### ISSUE THREAD

**Issue by Pingopete** [rbray89/EnvironmentalVisualEnhancements#33]
Since the ocean fixes update I have been unable to apply changes save values to cfg for the terrain shader alone, city lights manager and atmosphere (clouds and atmo) both still work as normal

**Comment by rbray89** [comment]
Are all values valid?

**Comment by Pingopete** [comment]
Yes I've tested with a stock install of the latest build including the new configs

**Comment by Pingopete** [comment]
After doing some further testing we found it to be associated with the ocean gui section alone, if i remove the ocean section it works fine, it can also make the whole globe one block colour

**Comment by Blacksuu** [comment]
after i show to Pete in live stream all glich how to be replicate & how it work in a temp fix i dig more in code & u found this https://github.com/rbray89/EnvironmentalVisualEnhancements/commit/7e21a9b9e682ee4ce8bac0e426479e6a41d37a05 what it means Ray you was change sistem to RGBA sistem, good idea,  BUT it sems to work partial & i was intrigue way eve it work fine if i make intentionaly to get a error, exception  aka i was smell some were in cod there are default value what work perfect = shader + each segment were you was set default value (i found them) & surprise explination came from https://github.com/rbray89/EnvironmentalVisualEnhancements/blob/Overhaul/ShaderLoader/Shaders/SphereOcean.shader#L3 for example > default value is working coz is old sistem :) = if anyone it make eve to get a error your cod activate rect sistem aka load default value

**Comment by Blacksuu** [comment]
p.s. if you want anytime to show you in a live stream what & how thing work just yell & we can arange that it will be greath if Nat or Blackrack will can be in same stream to see thing :) Pete i think he was found good thing after that :)
